sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -921,4 +923,55 @@ class MapboxMapTest {
     mapboxMap.clearData(callback)
     verify { Map.clearData(resourceOptions, callback) }
   }
+
+  @Test
+  fun setMapProjectionMercator() {
+    val mapProjection = MapProjection.Mercator
+    mapboxMap.setMapProjection(mapProjection)
+    verify {
+      nativeMap.mapProjection = Value.valueOf(
+        hashMapOf(
+          MapProjectionUtils.NAME_KEY to Value(
+            MapProjectionUtils.MERCATOR_PROJECTION_NAME
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun setMapProjectionGlobe() {
+    val mapProjection = MapProjection.Globe
+    mapboxMap.setMapProjection(mapProjection)
+    verify {
+      nativeMap.mapProjection = Value.valueOf(
+        hashMapOf(
+          MapProjectionUtils.NAME_KEY to Value(
+            MapProjectionUtils.GLOBE_PROJECTION_NAME
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun getMapProjection() {
+    mockkObject(MapProjectionUtils)
```

Comment:
if it's default check, should we mock utils class?

Reply:
it's not default as described above
we can discuss offline if we're on the same page